homeassistant/components/climate/__init__.py

```diff
@@ -933,6 +933,44 @@ async def async_service_temperature_set(
     entity: ClimateEntity, service_call: ServiceCall
 ) -> None:
     """Handle set temperature service."""
+    if (
+        ATTR_TEMPERATURE in service_call.data
+        and not entity.supported_features & ClimateEntityFeature.TARGET_TEMPERATURE
+    ):
+        report_issue = async_suggest_report_issue(
+            entity.hass,
+            integration_domain=entity.platform.platform_name,
+            module=type(entity).__module__,
+        )
+        _LOGGER.warning(
+            (
+                "%s::%s set_temperature action was used with temperature but the entity does not "
```

Comment:
We didn't say or add a comment when this will be removed. I think it's good to do that at least for our own memory.

Reply:
Good point as I search for those. Think it was missed for the previous one too so will make a PR to add that 👍

Reply:
Fixed with https://github.com/home-assistant/core/pull/125837